dashboards/google-microservices/microservices-grpc-monitoring-gce.json

```diff
@@ -92,7 +88,8 @@
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "timeSeriesQueryLanguage": "fetch gce_instance\n  | metric 'custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs'\n  | filter (metric.grpc_client_status != 'OK')\n  | align rate(1m)\n  | every 1m\n"
+                  "prometheusQuery": "rate(custom_googleapis_com:opencensus_grpc_io_client_completed_rpcs{monitored_resource=\"gce_instance\", grpc_client_status!=\"OK\"}[1m])\n",
+                  "unitOverride": "1/s"
```

Comment:
This has no complicated joins or aggregations... does this need to be PromQL? Or can this just be a simple query builder (aka ListTimeSeries) with a filter?